Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -113,7 +113,7 @@ - (void)initializeWithParameters:(id<MAAdapterInitializationParameters>)paramete
 {
     [self updateUserPrivacySettingsForParameters: parameters];
     
-    if ( [ALVungleInitialized compareAndSet: NO update: YES] )
+    if ( ![ALVungleInitialized get] && ALVungleIntializationStatus != MAAdapterInitializationStatusInitializing )
     {
```

Comment:
i think u dont need this ![ALVungleInitialized get] anymore. If Max is using it internally then u are setting it in the completion block so it should be fine.

Reply:
@ManojBudumuru-vungle would like to leave as is - 

- the ALVungleInitialized combined with the status check will allow the method to retry SDK init.
- If I remove the ALVungleInitialized getter, we'll attempt to init every time the status is != initializing (increase in init errors I'd like to avoid) - this method is called n+1 (at least) times on AL init and on every ad load event.
- If I leave as is, the bool flag, once set, will prevent method from calling SDK init unnecessarily (preventing a spike in init errors that might occur as mentioned in the second bullet).